tests/conftest.py

```
@@ -6,6 +6,36 @@
+    ----------
+    config : pytest.Config
+        The active pytest configuration (unused, required by the hook signature).
+
+    Returns
+    -------
+    list[str]
+        One-line-per-entry header additions.
+    """
+    try:
+        from boileroom.images.metadata import MODAL_IMAGE_TAG_ENV, get_docker_repository, get_modal_image_tag
+    except Exception as exc:  # pragma: no cover - defensive; keep pytest running if import fails
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)